File: platform/sdk/nodejs/examples/cloud/cloud-025_lifecycle_hot_to_warm_30d.js
const { CloudClient } = require("../../xuanji-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.createBucket("life-bucket");
const result = client.lifecycleHotToWarm30d("life-bucket");
if (!result.ok || result.rule.transition.days !== 30) process.exit(1);
console.log("XJ-OK: cloud-025_lifecycle_hot_to_warm_30d");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-025_lifecycle_hot_to_warm_30d.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.createBucket("life-bucket")
result = client.lifecycleHotToWarm30d("life-bucket")
if not result["ok"] or result["rule"]["transition"]["days"] != 30:
    sys.exit(1)
print("XJ-OK: cloud-025_lifecycle_hot_to_warm_30d")
sys.exit(0)
